bot/cogs/information.py

```diff
@@ -104,36 +100,18 @@ def __init__(self, bot: Bot):
         features = ", ".join(ctx.guild.features)
         region = ctx.guild.region
 
-        # How many of each type of channel?
         roles = len(ctx.guild.roles)
-        channels = ctx.guild.channels
-        text_channels = 0
-        category_channels = 0
-        voice_channels = 0
-        for channel in channels:
-            if type(channel) == TextChannel:
-                text_channels += 1
-            elif type(channel) == CategoryChannel:
-                category_channels += 1
-            elif type(channel) == VoiceChannel:
-                voice_channels += 1
+        member_count = ctx.guild.member_count
+
+        # How many of each type of channel?
+        channels = Counter({TextChannel: 0, VoiceChannel: 0, CategoryChannel: 0})
+        for channel in ctx.guild.channels:
+            channels[channel.__class__] += 1
```

Comment:
Use the `discord.ChannelType` enum instead of `__class__`. A channel's type can be retrieved with `channel.type`. `group` and `private` types should be ignored.

Reply:
Would this not ignore `discord.Category`? I could add an if statement for that specific type, but wouldn't that be less ideal than using `__class__`?

Reply:
In retrospect, I don't think any code needs to be written to ignore those types. Since it's only looking at guild channels, it should never encounter those channel types in the first place.